### PR TITLE
Add solara and solara-assets packages

### PR DESCRIPTION
**Pull request by giswqs**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

https://github.com/widgetti/solara/issues/105

### CODE REVIEW

**Comment by giswqs** [comment]
@conda-forge/help-python The recipes are ready for review. Thank you.

**Comment by maartenbreddels** [comment]
Great work @giswqs !
solara and solara-assets are always released together (with the same version number) and pinned to each other.
Would it be possible to have 1 recipe that will release both?
In my experience, releasing 2 separate packages on conda-forge with an interdependency is not great because it takes a few hours before the new release is seen. So you have to wait first, edit the recipe, then make sure CI is run again.

I tried to do something with voila once https://github.com/conda-forge/voila-feedstock/pull/51

**Comment by luk-f-a** [comment]
I don't have any experience with conda-forge, but reading the docs it might be possible, rihttps://conda-forge.org/docs/maintainer/adding_pkgs.html#adding-multiple-packages-at-once

File: recipes/flit-core/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{% set name = "flit-core" %}
```
Comment (bollwyvl) [inline recipes/flit-core/meta.yaml:1]: `flit-core` is provided by [`flit-feedstock`](https://github.com/conda-forge/flit-feedstock/blob/main/recipe/meta.yaml) multi-output recipe, which also demonstrates how `solora(-assets)` can be built in the same `meta.yaml`. All feedstocks -> packages can be seen on https://conda-forge.org/feedstock-outputs/

**Comment by dhirschfeld** [comment]
I missed this PR so ended up putting in one of my own - #23413.

That has been merged so now `solara` and `solara-assets` are being built from https://github.com/conda-forge/solara-feedstock:

```
❯ mamba search 'solara*'
Getting repodata from channels...

conda-forge/noarch                                  11.8MB @   9.9MB/s  1.2s
conda-forge/linux-64                                29.3MB @   5.2MB/s  5.7s


 Name          Version Build        Channel
────────────────────────────────────────────────
 solara        1.19.0  pyhd8ed1ab_0 conda-forge
 solara-assets 1.19.0  pyh1a96a4e_0 conda-forge
```

**Comment by giswqs** [comment]
Awesome! Closing this one.

**Pull request closed without merging**